filetracker/servers/files.py

```diff
@@ -55,6 +57,13 @@ def handle_PUT(self, environ, start_response):
         digest = environ.get('HTTP_SHA256_CHECKSUM', None)
         logical_size = environ.get('HTTP_LOGICAL_SIZE', None)
 
+        if compressed and digest and logical_size:
+            logger.debug('Handling PUT %s.', path)
```

Comment:
Shouldn't logged messages be in " instead of '? because theoretically they are translatable. 

Also https://github.com/sio2project/filetracker/pull/58/files#diff-5f132e5d465f2e3358b1ee84c313d907R52 uses double-quotes whereas other logs use single-quotes, so it would be good to at least have it consistent.

Reply:
I really don't thinking translating logs is a good idea: they're not supposed to be end-user facing, and using English everywhere is much more simpler for administration.

Reply:
Changed the other occurrence to single quotes.

Reply:
Yeah, I've never meant to suggest translating. 
I just think that our convention is to use ' for computer-facing strings (dict keys, config) and " for humand-facing strings - like error messages and logs.
But surely consistence is key, so I think single-quotes are ok.
@accek what do you think?

Reply:
Yeah, let's make it consistent.